source/transfer_list.rst

```diff
@@ -245,9 +245,9 @@ Anyone is free to use tags from that range for any custom TE layout without
 adding their definitions to this specification first. The use of this range is
 *strongly discouraged* for anything other than local experiments or code that
 will only ever be used in closed-source components owned by the entity
-controlling the entire final firmware image. In particular, a TE just
+controlling the entire final firmware image. In particular, a creator of a TE, just
 containing platform-specific data or internal structures specific to a single
-firmware implementation is no reason not to allocate a standardized tag for it
+firmware implementation, should allocate a standardized tag for it
```

Comment:
That makes it significantly weaker. Is there a way to encourage this?

Reply:
This reword was a nitpick.
It's simply removing the double negative.

We can drop the change.

I've reworded the sentence a bit in the latest push (hopefully respecting the original language):
"
In particular, a creator of a TE that just contains platform-specific data or internal structures specific to a single firmware implementation, should allocate a standardized tag for it in this specification -- using the non-standardized range is strongly discouraged.
"